test/test_singlelayerqg.jl

```diff
@@ -435,4 +435,28 @@ function test_1layerqg_enstrophy_drag(dev; deformation_radius=2.23)
   prob = SingleLayerQG.Problem(dev; nx=16, deformation_radius=deformation_radius)
   SingleLayerQG.enstrophy_drag(prob)
   return nothing
-end
\ No newline at end of file
+end
+
+"""
+    test_1layerqg_background_flow(dev; n, L, dt)
+
+Evolves a lamb vortex with a background flow such that the vortex remains stationary.
+"""
+function test_1layerqg_background_flow(dev::Device=CPU(); n=256, L=10, dt=0.01)
+	
+	prob = SingleLayerQG.Problem(dev; nx=n, Lx=L, dt, U = -1, stepper="FilteredRK4")
+	x, y = gridpoints(prob.grid)
+	
+	q₀ = lambdipole(1,1,prob.grid,center=(1e-10,0))
```

Comment:
```suggestion
	q₀ = lambdipole(1, 1, prob.grid, center=(1e-10, 0))
```

mind the spelling here

Reply:
:-D

https://www.sussex.ac.uk/informatics/punctuation/comma#:~:text=Each%20use%20has%20its%20own,followed%20by%20a%20white%20space.